src/VirtualClient/VirtualClient.Actions/Memtier/MemtierBenchmarkClientExecutor.cs

```diff
@@ -166,6 +166,17 @@ public int MaxClients
             }
         }
 
+        /// <summary>
+        /// Parameter defines the Delta.
+        /// </summary>
+        public int Delta
```

Comment:
what is delta? is this used?

Reply:
Sorry, I forgot to push the latest commit. This parameter defines the delta in terms of steps we increment for the spread of memtiercpuaffinity. The name I had in mind for this was MemtiercpuaffinityDelta, please suggest otherwise.